pkgs/applications/misc/keepassx/reboot.nix

```diff
@@ -1,23 +1,23 @@
 { stdenv, fetchFromGitHub, cmake, libgcrypt, qt5, zlib, libmicrohttpd, libXtst }:
 
 stdenv.mkDerivation rec {
-  name = "keepassx2-http-unstable-${version}";
-  version = "2016-05-27";
+  name = "keepassx-reboot-${version}";
+  version = "2.0.3-http";
```

Comment:
`2.0.3-http` is the name of the tag being fetched from GitHub. If the version name is going to be a problem, I can decouple the version from the tag name. If it's not an issue, it's probably good to remain consistent with the source repository.

Let me know how I should proceed here.